deploy/os/controller.yaml

```diff
@@ -51,16 +51,9 @@ spec:
               containerPort: 8443
           volumeMounts:
             - name: webhook-tls-certs
-              mountPath: /tmp/k8s-webhook-server/serving-certs
+              mountPath: /apiserver.local.config/certificates/
               readOnly: true
       volumes:
         - name: webhook-tls-certs
-          projected:
-            sources:
-              - configMap:
-                  name: devworkspace-controller-secure-service
-                  items:
-                    - key: service-ca.crt
-                      path: ./ca.crt
-              - secret:
-                  name: devworkspace-controller
+          secret:
+            secretName: workspace-controller
```

Comment:
```suggestion
            secretName: devworkspace-controller
```
?

Reply:
:+1: Changed here and in `service.yaml`.